README.md

```diff
@@ -20,12 +20,18 @@ Future features include the following:
 
 To use Kubedash, the following are required:
 - A Kubernetes cluster of v1.0 or higher is operational and accessible from kubectl.
-- Heapster v0.18.0 is running as the `heapster` service on the `kube-system` namespace.
+- For Kubernetes <1.2 Heapster v0.18.0 is running as the `heapster` service on the `kube-system` namespace.
```

Comment:
No. For >= 1.2 we don't care whether there is any Heapster or no. There is one in the bundle.